exchange/gdpr_test.go

```diff
@@ -148,6 +148,65 @@ func TestGetConsent(t *testing.T) {
 	}
 }
 
+func TestSelectEEACountries(t *testing.T) {
+	tests := []struct {
+		description         string
+		hostEEACountries    []string
+		accountEEACountries []string
+		expected            []string
+	}{
+		{
+			description:         "Account EEA countries provided, return account's",
```

Comment:
Nitpick: I would also shorten your names removing the expected return info. That can easily be deciphered. Simply describing the inputs is recommended.